mkdocs_include_markdown_plugin/event.py

```diff
@@ -54,10 +54,16 @@ def found_include_tag(match):
 
         return text_to_include
 
+    def interpret_escapes(str):
+        if str is None:
+            return None
```

Comment:
This should be taken out of the function and checked before. The function should only be run with strings as parameters.

Reply:
Are you comfortable then if I write in the main function:

`end = interpret_escapes(match.group('end') or '')`

or do you prefer:

```
end = match.group('end')
if end:
   end = interpret_escapes(end)
```

Reply:
We can avoid the call if doesn't match, so better like:

```python
end = match.group('end')
if end is not None:
   end = interpret_escapes(end)
```

Reply:
I left it as `if end:` because there is also no point in calling interpret_escapes on an empty string.

Reply:
I understand, but keep in mind that other developers could add in the future other processing here. So specify it as `is not None` is more readable and could potentially prevent errors if more processing is added without handle empty strings: you know in both cases that `None` is what will go here.

Reply:
Confused; what Python false-y value could you want to interpret escapes in? I don't understand how "is not None" is clearer than the less verbose simple test of whether it's a false-y or true-y value?  Of course I will check in whatever you want, so just tell me, but I don't understand how explicit comparison to None is better than the simple logical test, even in the context of future development. Thanks, Glen

Reply:
If a crazy developer in the future does not handle empty strings (`end` is empty string) your tests will fail because `interpret_escapes` will not be called. If `end` is an empty string your test would fail. Now it's impossible, because `end` only can be `None` or non-empty string, but this is not true in a potential future.

Reply:
Imagine that `end` could be `False`, 0 and other falsy values. Explicit is better than implicit.